### DISCUSSION TITLE
style:="transform: rotate(-90deg)"

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#15321] [How to]
Is it possible to rotate text in PDF template?

**Comment by mariuszkrzaczkowski** [comment]
looks like yes, I have not verified
https://github.com/YetiForceCompany/YetiForcePDF/search?q=rotate

**Comment by garryatbrighton** [comment]
yp.. .tried with inline css, but no turning
